v2/docs/gs/params.md

```diff
@@ -107,21 +124,27 @@ the KVDB until it reaches the maximum sustainable value for the underlying
 storage.  This ramp-up process can take up to **200 seconds**.
 
 For benchmarks, this initial throttling can *greatly* distort results.
-In general operation, this initial throttling may impact the time
+In normal use, this initial throttling may impact the time
 before a service is fully operational.
 
 The `throttling.init_policy` parameter can be used to achieve the maximum
 ingest rate in far less time.  It specifies a relative initial throttling
-value of `light` (minimum), `medium`, or `default` (maximum) throttling.
+value of `light` (minimum), `medium`, or `heavy` (maximum) throttling.
+
+If `throttling.init_policy` is set to `auto`, HSE currently selects (applies)
+`heavy` if the KVDB is configured with a capacity media class, and `light`
+if the KVDB is configured with only a pmem media class.
+The specific `throttling.init_policy` that HSE selects when `auto` is
+specified may change in future releases.
```

Comment:
1. No need to say "currently" in first sentence (b/c last sentence addresses that).

2. Same concern as previous comment regarding wording in last sentence: saying HSE selects throttling.init_policy in this context is mildy confusing.  Normally this wouldn't be confusing, but it feels self-referential here since this text is describing the parameter itself.  It sounds like HSE is changing the parameter value, but HSE is not doing that -- it is selecting the initial throttle setting.

Reply:
Agreed.  I reworked all of the auto-related paragraphs accordingly.